var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -56,7 +56,25 @@ def cmake_args(self):
     def setup_build_environment(self, env):
         env.set('PYBIND11_USE_CMAKE', 1)
 
+    def patch(self):
+        """ see https://github.com/spack/spack/issues/13559 """
+        filter_file('import sys',
+                    'import sys; return "{0}"'.format(self.prefix.include),
+                    'pybind11/__init__.py',
+                    string=True)
+
     def install(self, spec, prefix):
         super(PyPybind11, self).install(spec, prefix)
         setup_py('install', '--single-version-externally-managed', '--root=/',
                  '--prefix={0}'.format(prefix))
+
+    @run_after('install')
+    @on_package_attributes(run_tests=True)
+    def test(self):
+        with working_dir('spack-test', create=True):
+            # test include helper points to right location
+            python = Executable(self.spec['python'].command.path)
+            inc = python('-c',
+                         'import pybind11 as py; print(py.get_include())',
```

Comment:
This print syntax will only work in Python 2.6+. You should be able to do something like:
```python
'import pybind11 as py; ' + self.spec['python'].print_string('py.get_include()')
```

Reply:
That's okay, pybind11 only supports Python 2.7+ ;)
https://github.com/pybind/pybind11/#goodies